pkg/app/middleware.go

```diff
@@ -2,14 +2,23 @@ package app
 
 import (
 	"net/http"
+	"regexp"
 	"time"
 
+	"github.com/google/uuid"
 	"github.com/openshift/assisted-service/pkg/thread"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/rs/cors"
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	ipxeScriptQueryKey   = "file_name"
+	ipxeScriptQueryValue = "ipxe-script"
+)
+
+var ipxeScriptPattern = regexp.MustCompile(`^/api/assisted-install/v2/infra-envs/(?P<uuid>.*)/downloads/files`)
```

Comment:
I wish we could do something better than this but I don't see a good way :worried: